Hackney.Shared.HousingSearch.Tests/Domain/Tenure/TenureTests.cs

```diff
@@ -0,0 +1,19 @@
+using FluentAssertions;
+using Hackney.Shared.HousingSearch.Tests.Helper;
+using Xunit;
+
+namespace Hackney.Shared.HousingSearch.Tests
+{
+    public class TenureTests
+    {
+
+        [Fact]
+        public void TenureHasPropertiesSet()
```

Comment:
is this testing the helper rather than the new implementation?

Reply:
Moved to different approach.